skimage/util/montage.py

```diff
@@ -108,3 +121,68 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, padding_width=0):
```

Comment:
Sounds a great idea, bit I'm really off these nD discussions as I never use this feature at all.